client/blacklist.py

```diff
@@ -0,0 +1,69 @@
+from flask import current_app
+from netaddr import IPNetwork, IPAddress
+from netaddr.core import AddrFormatError
+
+
+bl_request_ip = []  # array of tuples (network mask, port)
+bl_website_ip = []  # array of tuples (network mask, port)
+
+
+def build_inc_request_blacklist(logger):
+    with open("config/blacklist_inc_request_ip.txt") as f:
```

Comment:
Should we give some flexibility as to where they want to store this information? They might not want to push a file containing their blacklists. Maybe accept the file name as a config parameter instead or something?